airflow-core/src/airflow/ui/src/components/Clear/TaskInstance/ClearTaskInstanceDialog.tsx

```diff
@@ -105,15 +112,19 @@ const ClearTaskInstanceDialog = ({ onClose, open, taskInstance }: Props) => {
               multiple
               onChange={setSelectedOptions}
               options={[
-                { disabled: taskInstance.logical_date === null, label: "Past", value: "past" },
-                { disabled: taskInstance.logical_date === null, label: "Future", value: "future" },
+                { disabled: !Boolean(logicalDate), label: "Past", value: "past" },
+                { disabled: !Boolean(logicalDate), label: "Future", value: "future" },
                 { label: "Upstream", value: "upstream" },
                 { label: "Downstream", value: "downstream" },
                 { label: "Only Failed", value: "onlyFailed" },
               ]}
             />
           </Flex>
-          <ActionAccordion affectedTasks={affectedTasks} note={note} setNote={setNote} />
+          <ActionAccordion
+            affectedTasks={affectedTasks}
+            note={taskNote === undefined ? undefined : note}
```

Comment:
I briefly thought about if it would make sense to keep the notes fied and if all are cleared then the note is applied to all mapped tasks?
But not critical / blocking. Can also be adjusted in future.

Reply:
Mark state as and setting a note are the same patch endpoint which doesn't work right now.

Reply:
If I remember correctly you need to pass a tuple somewhere.

I’ll double check tomorrow. Otherwise I think we should fix the backend

Reply:
Oh that’s the bulk patch ? Anyway I need to check the code tomorrow

Reply:
Added to my todo list, I'll work on the backend part soon.